automation_tools/__init__.py

```diff
@@ -1722,19 +1730,25 @@ def katello_installer(debug=False, sam=False, verbose=True, **kwargs):
     True
 
     """
-    # capsule-dns-forwarders should be repeated if setting more than one value
-    # check if a list is being received and repeat the option with different
-    # values
+    # capsule-dns-forwarders should be repeated if setting more than one
+    # value check if a list is being received and repeat the option with
+    # different values
+    proxy = 'capsule'
+    installer = 'katello'
+    if os.environ.get('SATELLITE_VERSION') == '6.2':
```

Comment:
We should leave this helper function as a helper function, that said environment variables should be handled outside. Please add a parameter called satellite_version (default to `'6.2'`?) and handle the environment variable on product_install, also product_install should fail early if that variable is not defined.